great_expectations/datasource/fluent/data_asset/data_connector/file_path_data_connector.py

```diff
@@ -148,15 +148,15 @@ def get_data_reference_count(self) -> int:
 
     # Interface Method
     @override
-    def get_matched_data_references(self) -> List[str]:
```

Comment:
this change is required to test the connection of a BatchDefinition. The other interface methods adjacent to this will eventually require either a regex or a batch request as well.